tests/test_buffer.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
```

Comment:
Add a test for the possible exceptions? e.g.
  - `ValueError` in `sample` when buffer is empty
  - `ValueError` in `store` if `data` is empty, wrong shape, or too large.

If easy, also add a test for `fromData`.